metadata.json

```diff
@@ -13,6 +13,10 @@
   ],
   "dependencies": [
     {
+      "name": "puppetlabs/stdlib",
+      "version_requirement": ">= 4.9.0 < 5.0.0"
+    },
+    {
       "name": "simp/simplib",
       "version_requirement": ">= 2.0.0 < 3.0.0"
```

Comment:
This dependency (and the one in requires) should be updated to 2.1.0 since it has a validation on a new simplib type, EmailAddress